app/code/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsert.php

```diff
@@ -13,17 +13,25 @@ class OnInsert extends \Magento\Cms\Controller\Adminhtml\Wysiwyg\Images
      */
     protected $resultRawFactory;
 
+    /**
+     * @var \Magento\Cms\Model\Wysiwyg\Images\PrepareImage
+     */
+    protected $prepareImage;
+
     /**
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\Framework\Registry $coreRegistry
      * @param \Magento\Framework\Controller\Result\RawFactory $resultRawFactory
+     * @param \Magento\Cms\Model\Wysiwyg\Images\PrepareImage $prepareImage
      */
     public function __construct(
         \Magento\Backend\App\Action\Context $context,
         \Magento\Framework\Registry $coreRegistry,
-        \Magento\Framework\Controller\Result\RawFactory $resultRawFactory
+        \Magento\Framework\Controller\Result\RawFactory $resultRawFactory,
+        \Magento\Cms\Model\Wysiwyg\Images\PrepareImage $prepareImage
```

Comment:
Please add the constructor dependency in a backwards-compatible way.

See "Adding a constructor parameter" section of https://devdocs.magento.com/contributor-guide/backward-compatible-development/